docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
       - REDIS_URL=redis://redis:${REDIS_PORT}
       # variables for elasticsearch
       - ES_HOST=elasticsearch
-      - ELASTIC_URL=http://elasticsearch:${ELASTIC_PORT}
+      - ELASTIC_URL=http://elasticsearch
```

Comment:
Why this change?  If I want to run es natively, or over the network, why not allow that?

Reply:
Was this not the cause of the issue according to @manekenpix